### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DATAOORTS**
Added Dataoorts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Ever had a laptop 💻 which is not powerful enough to run your models , forget
 | RunPod | https://www.runpod.io/gpu-instance | [pricing :label: ](https://www.runpod.io/gpu-instance/pricing) | - |
 | TensorDock | https://tensordock.com | [pricing :label: ](https://tensordock.com/pricing) [pricing :label:](https://console.tensordock.com/deploy) | Discounts to FOOS, students and researchers |
 | Vast.ai | https://vast.ai | [pricing :label: ](https://vast.ai/console/create/) | - |
+| [Dataoorts](https://dataoorts.com/) | https://dataoorts.com/ | [pricing :label: Starting at $0.55/hr for 1x A100, $0.69/hr for 1x H100 and $0.88 for 1x GH200](https://dataoorts.com/pricing) | - |
```
Comment (InCogNiTo124) [inline README.md:47]: Thank you for the contribution! Since we keep the alphabetical ordering, could you move this below dataiku (line 24)?

**Review by InCogNiTo124 — CHANGES_REQUESTED** [review]

**Review by DATAOORTS — COMMENTED** [review]
Just Update it!

**Comment by Curlypla** [comment]
The site gives very little confidence, all is empty, I do not say that it is a scam but I don't know if it is a good idea to put it without having proven itself.

**Comment by InCogNiTo124** [comment]
Thanks for raising concerns @Curlypla.

I did my research about DATAOORTS. I am not sure what you mean by "all is empty" since they seem to have enough content, but it _did_ strike me how unoptimized their page was (slow loads, broken CSS), and how many typos they have.

That said, I concluded that DATAOORTS is probably a new ML platform startup that tries to obtain their first customers, and _I have nothing against that_. This repo is a list of cloud deep learning platforms, but **not** a curated one. If they truly offer a service that is better than the rest, this will enable them to prove themselves. If they don't, there is a lot of other entries in that list for a user to pick from :))

@zszazi do you have anything to add?

**Comment by rajat709** [comment]
Hi @Curlypla , I understand that there is currently limited content. However, I'm highly value your feedback, as it will guide us in adding more relevant material in the future. We believe in the importance of concise and meaningful content over an abundance of words that lack substance. Our primary focus is on prioritizing concepts and solutions that we have been diligently working on for the past five months. After continuous testing, we are now confident in moving forward to production, with regular updates planned.

**Comment by rajat709** [comment]
Thank You @InCogNiTo124 / cc @DATAOORTS | We should take note of this!

**Pull request merged**